src/gui_common/tooltip/ToolTipManager.tscn

```diff
@@ -342,7 +342,7 @@ margin_bottom = 20.0
 mouse_filter = 2
 script = ExtResource( 21 )
 DisplayName = "STORAGE"
-ModifierValue = "+1"
+ModifierValue = "+0.5"
```

Comment:
Regarding translations this is a bit problematic as at least for finnish translation this should turn into "+0,5"...
I'll accept these changes for now and open a new issue about that.